src/pages/Users/components/ButtonsGroup/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import BoxBase from "src/components/Boxs/BoxBase";
+import ButtonBase from "src/components/Buttons/ButtonBase";
+
+export interface IActionProps {
+    userId: string;
+    isActive: boolean;
+    onUpdate: (userId: string) => void;
+    onChangeStatus: (userId: string) => void;
```

Comment:
I think we should get the interface for userId from Users["userId"] in the generated file

Reply:
![image](https://user-images.githubusercontent.com/58205909/133921589-6dffe7fc-1dc9-4d9e-8601-074ccd307fc5.png)

Reply:
:thinking: